packages/cli/openapi-parser/src/options.ts

```diff
@@ -9,11 +9,14 @@ export interface ParseOpenAPIOptions {
     useTitlesAsName: boolean;
     /* The audiences that the OpenAPI must be filtered down into */
     audiences: string[] | undefined;
+    /* TODO: pick a better name for this flag, and maybe move it to AsyncAPI-only config */
+    useImprovedMessageNaming: boolean;
```

Comment:
yup would be great to put this in a `ParseAsyncAPIOptions` and expose that as well

Reply:
`naming: v2` or something ?